pandas/tests/reshape/concat/test_empty.py

```diff
@@ -249,3 +249,18 @@ def test_empty_dtype_coerce(self):
         result = concat([df1, df2])
         expected = df1.dtypes
         tm.assert_series_equal(result.dtypes, expected)
+
+    def test_concat_empty_dataframe(self):
+        # 39037
+        df1 = DataFrame(columns=["a", "b"])
+        df2 = DataFrame(columns=["b", "c"])
+        result = concat([df1, df2, df1])
+        expected = DataFrame(columns=['a', 'b', 'c'])
```

Comment:
to standardise, we all use double quote, and I think pre-commit job will fail on this